### PR TITLE
include bash

### PR DESCRIPTION
**Pull request by willscott**
per https://stackoverflow.com/questions/61039877/add-shell-or-bash-to-a-docker-image-distroless-based-on-debian-gnu-linux

### CODE REVIEW

**Review by hannahhoward — APPROVED** [review]
LGTM

**Comment by masih** [comment]
I'm curious why this is needed. 

If needed i recommend using distriless debug base images.

**Comment by willscott** [comment]
@masih  when arguments are included in the `entrypoint` command of the dockerfile, docker transparently rewrites the entrypoint of the final image to `/bin/sh -c <entrypoint>`

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,11 @@ RUN go mod download
...
 
+FROM busybox:1.35.0-uclibc as busybox
+
 FROM gcr.io/distroless/static-debian11
 COPY --from=build /go/bin/recorder /usr/bin/
+COPY --from=busybox /bin/sh /bin/sh
 
-ENTRYPOINT ["/usr/bin/recorder", "-mongoDB", "SP Reputation", "-mongoCollection", "lassie" "-mongoPercent", "0.01"]
+ENTRYPOINT ["/usr/bin/recorder"]
+CMD ["-mongoDB", "SP Reputation", "-mongoCollection", "lassie" "-mongoPercent", "0.01"]
```
Comment (masih) [inline Dockerfile:18]: Missing comma after `"lassie"`.

**Pull request merged**